test/functional/feature_block.py

```diff
@@ -83,6 +83,7 @@ def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
         self.extra_args = [['-acceptnonstdtxn=1']]  # This is a consensus block test, we don't care about tx policy
+        self.rpc_timeout = 1920
```

Comment:
in commit b737e5397af4cedde1aec47f5b4e87c2521efdd0

Out of scope for this pr, but I am wondering if it wouldn't be easier to compile an increased rpc timeout factor into the test config file. This way, all tests are updated when compiled with ipc and it would avoid having to update them manually and individually.

Reply:
> in commit [b737e53](https://github.com/bitcoin/bitcoin/commit/b737e5397af4cedde1aec47f5b4e87c2521efdd0)
> 
> Out of scope for this pr, but I am wondering if it wouldn't be easier to compile an increased rpc timeout factor into the test config file. This way, all tests are updated when compiled with ipc and it would avoid having to update them manually and individually.

That's a good idea, I didn't know about the timeout factor and it'd probably better to increase that if there are other timeouts outside this test. Some IPC performance problems are actually very easy to fix after you know they exist, though, so there might be some argument for increasing timeouts selectively instead of globally to identify these places, at least to start out with before it becomes a burden.